css/details.css

```diff
@@ -12,13 +24,143 @@ th {
 }
 
 td {
-	text-align: right;
+	text-align: left;
+	padding-left: 1em;
+	max-width: 4em;
 }
 
 summary {
 	cursor: pointer;
+	margin-bottom: 1em;
+	user-select: none;
+}
+
+/* Imprecise rows */
+
+tr.imprecise {
+	font-style: italic;
+	z-index: 10;
+}
+
+tr.imprecise td {
+	position: relative;
+}
+
+/* Question mark circle icon */
+tr.imprecise td::after {
+	content: "?";
+	position: absolute;
+	top: 0.65em;
+	right: -1.2em;
+	width: 1.2em;
+	height: 1.2em;
+	border-radius: 1em;
+	border: 1px solid #333;
+	font-size: 90%;
+	font-style: normal;
+	color: #000;
+	line-height: 15px;
+	font-weight: bold;
+	text-align: center;
+	background: #fff;
+	cursor: pointer;
+}
+
+
+/* Explanation tooltip for cotisation lines */
+
+.imprecise td aside {
```

Comment:
Use consistent selectors! By using sometimes `tr.imprecise` and sometimes `.imprecise`, you're making the specificity unreliable. If it's voluntary, then put the less specific instructions above the more specific ones.